backend/app/services/chat/trigger/emitter.py

```
@@ -0,0 +1,446 @@
+        """Initialize FlowEventEmitter.
+
+        Args:
+            execution_id: The FlowExecution ID to update
+        """
+        self.execution_id = execution_id
+
+    async def emit_chat_done(
+        self,
+        task_id: int,
+        subtask_id: int,
+        offset: int,
```

Ruff (0.14.10)

Unused method argument: `offset`

(ARG002)